usingr.Rmd

```diff
@@ -0,0 +1,359 @@
+pkgs <- c("dplyr", "knitr")
+x<-lapply(pkgs, library, character.only = TRUE)
+opts_chunk$set(tidy = FALSE, message = F, warning = F)
+```
+
+# Using R for bioassessment data: What, why, and how
+
+## Lesson Outline
+
+- [Goals and Motivation]
+- [RStudio]
+- [R language fundamentals]
+- [Data structures in R]
+- [Data I/O]
+
+## Lesson Exercises
+
+- [Exercise 1]
+- [Exercise 2]
+- [Exercise 3]
+
+## Goals and Motivation
+
+[R](https://www.r-project.org/) is a language for statistical computing as well as a general purpose programming language. Increasingly, it has become one of the primary languages used in data science and for data analysis across many of the natural sciences. 
+
+The goals of this training are to expose you to fundamentals and to develop an appreciation of what's possible with this software.  We also provide resources that you can use for follow-up learning on your own. You should be able to answer these questions at the end of this session:
+
+* What is R and why should I use it?
+* Why would I use RStudio and RStudio projects?
+* How can I write, save, and run scripts in RStudio?
+* Where can I go for help?
+* What are the basic data structures in R?
+* How do I import data? 
+
+### Why should I invest time in R?
+
+There are many programming languages available and each has it's specific benefits. R was originally created as a statistical programming language but now it is largely viewed as a 'data science' language. Why would you invest time in learning R compared to other languages?  
+
+* The growth of R as explained in the [Stack Overflow blog](https://stackoverflow.blog/2017/10/10/impressive-growth-r/), [IEEE rating](http://blog.revolutionanalytics.com/2016/07/r-moves-up-to-5th-place-in-ieee-language-rankings.html)
+
+R is also an open-source programming language - not only is it free, but this means anybody can contribute to it's development. As of `r format(Sys.time(), "%Y-%m-%d")`, there are `r nrow(available.packages())` on CRAN!   
+
+## RStudio
+
+In the old days, the only way to use R was directly from the Console - this is a bare bones way of running R only with direct input of commands. Now, [RStudio](https://www.rstudio.com/) is the go-to Interactive Development Environment (IDE) for R. Think of it like a car that is built around an engine. It is integrated with the console (engine) and includes many other features to improve the user's experience, such as version control, debugging, dynamic documents, package manager and creation, and code highlighting and completion. 
+
+Let's get familiar with RStudio before we go on.  
+
+### Open R and RStudio
+
+If you haven't done so, download and install RStudio from the link above.  After it's installed, find the RStudio shortcut and fire it up (just watch for now).  You should see something like this: 
+
+![](figure/rstudio.png)
+
+There are four panes in RStudio: 
+
+* Source
+* console
+* Environment, History, etc.
+* Files, plots, etc.
+
+### RStudio projects
```

Comment:
This is good, will you talk about folder structures/management within this section? Should we explicitly add some content/challenge asking folks to create "data", "scripts" folders and create/name a script "learning_r_at_CABW.R".

Reply:
Yep, this is covered a bit in exercise one.  I also made a note to explicitly call out data/file strucure in exercise 3.